blueocean-pipeline-api-impl/src/main/java/io/jenkins/blueocean/rest/impl/pipeline/PipelineNodeImpl.java

```diff
@@ -243,6 +240,18 @@ public HttpResponse restart(StaplerRequest request) {
         return null;
     }
 
+    protected static WorkflowRun getRun(WorkflowJob job, long itemId) throws InterruptedException {
+        WorkflowRun restartRun = QueueUtil.getRun(job, itemId);
+        if (restartRun == null) {
+            long startTimeMs = System.currentTimeMillis();
+            while (restartRun == null && System.currentTimeMillis()-startTimeMs < 1000) {
```

Comment:
could we make this `1000` configurable? like ` Integer.getInteger("wait.job.inqueue", 1000) `

Reply:
Done